crates/polars-arrow/src/io/avro/write/schema.rs

```diff
@@ -7,15 +7,15 @@ use polars_error::{polars_bail, PolarsResult};
 use crate::datatypes::*;
 
 /// Converts a [`ArrowSchema`] to an Avro [`Record`].
-pub fn to_record(schema: &ArrowSchema) -> PolarsResult<Record> {
+pub fn to_record(schema: &ArrowSchema, name: &str) -> PolarsResult<Record> {
```

Comment:
Can we accept a string here?

Reply:
Thanks @ritchie46 - the reason was with `String` I was getting:
```rust
  --> /Users/user/git/polars/crates/polars-io/src/avro/write.rs:65:48
   |
65 |         let record = write::to_record(&schema, self.name)?;
   |                                                ^^^^^^^^^ move occurs because `self.name` has type `std::string::String`, which does not implement the `Copy` trait
```
But I just realized `self.name.clone()` fixes this.

Is this the right approach?

Reply:
Yeap.